modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -59,3 +59,41 @@ In Go versions 1.13 and later, TCP is used automatically if DNS resolution using
 musl-based containers, including Alpine-based containers, do not support the `use-vc` option.
 ====
 
+[discrete]
+[id="openstack-upgrade-limitations_{context}"]
+== {rh-openstack} upgrade limitations
+
+There are limitations to consider when using Kuryr SDN during the {rh-openstack}
+upgrades process, in relation to Octavia and its load balancers updates.
+
+When OpenStack is upgraded, besides the expected Octavia's API changes, there is
+a need for updating the amphora image that will be used for the load balancer
+VMs. However, there is two options for the existing load balancer VMs:
+
+* {rh-openstack} operator handles the image upgrade of the existing load balancer VMs by triggering a link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/networking_guide/sec-octavia#update-running-amphora-instances[Load Balancer failover] action per existing Load Balancer.
+* {rh-openstack} operator does nothing related to existing load balancers VMs
+and leaves the update for the users.
+
+Both actions have implications when using Kuryr SDN:
+* If the {rh-openstack} operator handles the upgrade, there may be problems
+related to Network Policies enforcement when upgrading from {rh-openstack}
+versions before 16 to {rh-openstack} 16. The main reason is that Octavia
+provides in {rh-openstack} version 16 a new API to handle the access
+restriction to the Load Balancer in a standard way. This is incompatible with
+the previous mechanism that Kuryr uses to restrict the access in {rh-openstack}
+versions earlier than 16. Since Kuryr SDN has no visibility on when the failover
+action is triggered, the existing Security Group rules will be deleted during
+the upgrade and existing Network Policies associated to the related
+Services/Load Balancers will not be enforced. This requires either a Service or
+a Network Policy recreation by the user. It is therefore recommended that the
+{rh-openstak} Operator does not upgrade the existing Load Balancers for the
+{product-title} when Network Policies are used.
+
+* If the {rh-openstack} operator does not handle the upgrade of existing Load
+Balancer VMs, new features enabled by Octavia API (e.g., UDP listeners) won't
+work on the existing load balancers. Given that UDP services where not supported
+in {rh-openstack} versions earlier than 16, if Kuryr SDN is used and the
+operators is not handling the load balancer upgrades, users need to recreate
+their Services if they want to expose UDP ports, instead of modifying the
+existing ones to add UDP ports.
```

Comment:
perhaps better to highlight that it's a modification on the service specification